### DISCUSSION TITLE
The author’s 2 months of effort

### DISCUSSION THREAD

**Discussion by danceking** [ChefKissInc/NootedRed#169] [Show and tell]
In 2 months, for each version, geekbench6 test screenshots, 334dc21 version has greatly improved！！！
Great project, thank you to the author for your hard work! ! !

![cb0e2fd](https://github.com/ChefKissInc/NootedRed/assets/17052878/d200a817-2923-4c55-b529-2c711fde793f)
![60abe55](https://github.com/ChefKissInc/NootedRed/assets/17052878/9cc5babf-4b53-411c-81c7-73accc7c2082)
![d0b87b3](https://github.com/ChefKissInc/NootedRed/assets/17052878/5601e302-f8da-4ebd-af0e-02c1b4bc2a73)
![cfa68b9](https://github.com/ChefKissInc/NootedRed/assets/17052878/dee631a7-5361-42ad-89a3-26342660abce)
![6e61dc9](https://github.com/ChefKissInc/NootedRed/assets/17052878/61d8c9e1-5c3d-48d8-8d1f-25113f7c4cf6)
![189b852](https://github.com/ChefKissInc/NootedRed/assets/17052878/7de35ed4-b643-4d3f-9623-9afd3974cbf5)
![ede4b28](https://github.com/ChefKissInc/NootedRed/assets/17052878/f9b33ed2-001f-49d1-90c1-4965c3251db4)
![334dc21](https://github.com/ChefKissInc/NootedRed/assets/17052878/c1b6c12d-3a19-4b10-afee-0001ddd66d51)

**Comment by VisualEhrmanntraut** [comment]
You're welcome. Please use the Discussions tab next time, as you're not filing a bug or request. There's even a nice "Show and Tell" category. I've converted the issue to a discussion for you.

**Reply by danceking** [reply]
Sorry, I don't know much about github.

**Comment by danceking** [comment]
commit:334dc21
After testing most websites and video playback on edge, hardware acceleration is turned on normally.
However, I found that website videos cannot be played in full screen (such as YouTube) and will get stuck. Press esc to exit full screen and you can continue to watch the video.

**Comment by VisualEhrmanntraut** [comment]
@danceking Tip: Remove `AMDRyzenCPUPowerManagement` & `SMCAMDProcessor` for better performance and temperatures. Add `SMCProcessorAMD` for readings.

**Reply by danceking** [reply]
good tip!!!
![334dc21_add](https://github.com/ChefKissInc/NootedRed/assets/17052878/562a5bf9-8f59-44b3-aab8-bce099e0e394)
